docs/README.md

```diff
@@ -441,6 +441,90 @@ You can expand any of the messages that matches the filter to see the full stack
 
 ![image](./images/user-guide/35-logdet.png)
 
+### SLIs, SLOs and Burn rate Alerts
+
+Cloud operations sandbox comes with several predefined SLOs(Service level objectives), that allow us to measure our users happiness. To learn more about SLIs and SLOs [visit here.](https://cloud.google.com/blog/products/devops-sre/sre-fundamentals-slis-slas-and-slos)
+
+Cloud operations suite provides **service oriented monitoring**, that means that we are configuring SLIs, SLOs and Burning Rates Alerts for a 'service'.  
+
+The first step in order to create SLO is to **ingest the data**, for GKE services  it comes out of the box, but you can also ingest additional data and [create custom metrics.](#Monitoring-and-logs-based-metrics)
```

Comment:
nit: the , should be a period after "ingest the data". Also, you should specify what you mean by "it comes out of the box". (do you mean logs are captured?)

Reply:
replaced with telemetry and dashboards. is it clearer?

Reply:
yeah, that looks great